### PR TITLE
Add certificate info

### PR DESCRIPTION
**Pull request by djmagee**
## What does this PR do?

- Add "How to manage certificates in Temporal Cloud"
- Move upgrade policy from the Cloud landing page to its own page
- Add concept pages for Cloud Account Id, Cloud Namespace Name, and Cloud Namespace Id

## Preview

**https://deploy-preview-1330--mystifying-fermi-1bc096.netlify.app/cloud**

### CODE REVIEW

File: docs/cloud-context/upgrade-policy-for-temporal-cloud.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+id: upgrade-policy-for-temporal-cloud
+title: What is the upgrade policy for Temporal Cloud?
+sidebar_label: Upgrade policy
+description: Basic upgrade policy for customers of Temporal Cloud
+---
+
+All customers will automatically be upgraded to the latest minor version.
+
+Maintenance support, for major versions, will be provided for at least 18 months after a GA release, and there will be at least 6 months' notice before EOL/deprecating support.
```
[the remark this answers is not in the document]
Comment (djmagee) [reply docs/cloud-context/upgrade-policy-for-temporal-cloud.md:10]: However, in https://developers.google.com/style/numbers?hl=en#numbers-as-numerals, see "...use numerals for...Numbers less than 10 when they appear in the same sentence with numbers greater than 9."

File: docs/concepts/what-is-a-cloud-namespace-name.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+---
+
+A Cloud Namespace Name is a customer-supplied name for a [Namespace](/namespaces) in Temporal Cloud.
+Each Namespace Name, such as `accounting-production`, is unique within the scope of a customer's account.
+It cannot be changed after the Namespace is provisioned.
+
+Each Namespace Name must conform to the DNS naming rules:
+
+- A Namespace Name must begin or end with a letter or number and can contain only letters, numbers, and the dash (-) character.
+- Each dash (-) character must be immediately preceded and followed by a letter or number; consecutive dashes are not permitted.
```
[the remark this answers is not in the document]
Comment (djmagee) [reply docs/concepts/what-is-a-cloud-namespace-name.md:17]: Yep—should be "hyphen" throughout.

File: docs/cloud/how-to-manage-certificates-in-temporal-cloud.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+- Each certificate in the bundle must be either a root certificate or issued by another certificate in the bundle.
+- Each certificate in the bundle must include `CA: true`.
+- A certificate cannot be a well-known CA (such as DigiCert or Let's Encrypt) _unless_ the user also specifies certificate filters.
+- The signing algorithm must include SHA-256 or stronger. SHA-1 and MD5 signing algorithms are rejected.
+
+## Generate CA and client certificates
+
+Temporal Cloud authenticates a client connection by validating the client certificate against one or more CA certificates that are configured for the specified Namespace.
+
+### You already have certificate management infrastructure
```
Comment (mnichols) [inline docs/cloud/how-to-manage-certificates-in-temporal-cloud.md:48]: The `You already have` and `You have no` sections are really two different paths but presented flat like this, that isn't very clear. 
As a fast-follow iteration (not in this PR) maybe we could present them as "Options" or something to make that a little more like a decision tree.

**Review by mnichols — APPROVED** [review]
LGTM thanks for bringing it together!

File: docs/cloud/how-to-manage-certificates-in-temporal-cloud.md (hunk 3 above)
Comment (djmagee) [reply docs/cloud/how-to-manage-certificates-in-temporal-cloud.md:48]: Agreed. This one has been nibbling at me for a while.

**Pull request merged**